test/commonkafka.py

```diff
@@ -68,20 +75,23 @@ def update_kafka_connector(setup, params):
     logger.error("Failed to update connector, response code - ", response.status_code)
     return False
 
-def delete_kafka_connector(setup, params):
+
+def delete_kafka_connector(setup, connector):
     '''
     Delete kafka connect connector using kafka connect REST API
     '''
-    response = requests.delete(url=setup["kafka_connect_url"] + "/connectors/" + params["name"],
-                        headers={'Accept': 'application/json', 'Content-Type': 'application/json'})
-
+    if not isinstance(connector, str):
+        connector = connector['name']
+    response = requests.delete(url=setup["kafka_connect_url"] + "/connectors/" + connector,
+                               headers={'Accept': 'application/json', 'Content-Type': 'application/json'})
     if response.status_code == 204:
-        logger.info("Deleted connector successfully - " + json.dumps(params))
+        logger.info("Deleted connector successfully - {}".format(connector))
```

Comment:
Can you change the formatting style for all the logs in this file? If it's being changed here, all the logs in the file should be updated to reflect similar style.